session6/src/main.py

```diff
@@ -0,0 +1,118 @@
+import sys
+import pygame
+
+from platform import Platform
+
+from player import Player
+
+"""
+SETUP section - preparing everything before the main loop runs
+"""
+pygame.init()
+
+# Global constants
+SCREEN_WIDTH = 1000
+SCREEN_HEIGHT = 800
+FRAME_RATE = 60
+
+# Useful colors
+# format is (R,G,B) where each value is 0-255
+BLACK = (0, 0, 0)
+WHITE = (255, 255, 255)
+VIOLET = (139, 7, 54)
+
+# Initialized the rectangle
+rectangle = pygame.Rect(50, 50, 100, 100)
+
+# Creating the screen and the clock
+screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
+screen.set_alpha(0)  # Make alpha bits transparent
+clock = pygame.time.Clock()
+
+# Create our platforms, passing in x,y,width,height
+platform = Platform(300, 700, 200, 50)
+platform1 = Platform(600, 600, 200, 50)
```

Comment:
Personal preference - don't name one variable `platform` and the other one `platform1`. Would rather do `platform1` and `platform2`
```suggestion
platform1 = Platform(300, 700, 200, 50)
platform2 = Platform(600, 600, 200, 50)
```